modules/core/04-channel/keeper/upgrade.go

```diff
@@ -86,7 +86,7 @@ func (k Keeper) WriteUpgradeTryChannel(
 }
 
 // constructProposedUpgrade returns the proposed upgrade from the provided arguments.
-func (k Keeper) constructProposedUpgrade(ctx sdk.Context, portID, channelID string, fields types.UpgradeFields, timeout types.UpgradeTimeout) (types.Upgrade, error) {
+func (k Keeper) constructProposedUpgrade(ctx sdk.Context, portID, channelID string, fields types.UpgradeFields, timeout types.Timeout) (types.Upgrade, error) {
```

Comment:
minor consistency nit:
```suggestion
func (k Keeper) constructProposedUpgrade(ctx sdk.Context, portID, channelID string, fields types.UpgradeFields, upgradeTimeout types.Timeout) (types.Upgrade, error) {
```